src/Main.elm

```diff
@@ -211,26 +214,19 @@ view model =
         ]
 
 
-renderCardSetRadios : String -> List (Html Msg)
-renderCardSetRadios selectedCardSet =
+renderCardSetList : String -> List (Html Msg)
+renderCardSetList selectedCardSet =
     List.map
-        (\{ title, file } ->
-            let
-                titleId =
-                    String.toLower (String.replace " " "" title)
-            in
-            [ input
-                [ type_ "radio"
-                , onClick (SelectedCardSet file)
-                , checked (file == selectedCardSet)
-                , id titleId
+        (\{ title, iconSrc, file } ->
+            li []
+                [ a
```

Comment:
These aren't links, they don't go anywhere.

Reply:
I've changed them to button as that was easier than reverting to checkboxes. What they do is reload the page with the associated query param. The re-render is a kind of annoying side-effect. They way it is set up right now, we'd have to use the same technique to load the easy/med/hard version of the set one selected.

Reply:
If we want to keep both of the set and difficult controls on one page, it's probably two radio button groups and a submit.

A lot of games keep one choice per screen, I think this can often be to keep controls nice and chunky (but also good general design pattern!). If we wanted to do this and update app state then that would just be a button, but using links might be useful for maintaining user expectations of how a back button should work.

Reply:
OK I guess we should have a quick chat about it before I implement? I'd like to do it today maybe... but I can try with just adding radios.